operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -3810,6 +3810,129 @@
       },
       "id": 79,
       "panels": [
+        {
+          "description": "Watcher work queue depth",
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "fixedColor": "green",
+                "mode": "thresholds"
+              },
+              "mappings": [],
+              "noValue": "No Data",
+              "thresholds": {
+                "mode": "absolute",
+                "steps": [
+                  {
+                    "color": "green",
+                    "value": null
+                  }
+                ]
+              },
+              "unit": "none"
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 6,
+            "w": 3,
+            "x": 0,
+            "y": 67
+          },
+          "id": 476,
+          "options": {
+            "colorMode": "value",
+            "graphMode": "area",
+            "justifyMode": "auto",
+            "orientation": "auto",
+            "reduceOptions": {
+              "calcs": [
+                "lastNotNull"
+              ],
+              "fields": "",
+              "values": false
+            },
+            "text": {},
+            "textMode": "auto"
+          },
+          "pluginVersion": "9.5.17",
+          "targets": [
+            {
+              "editorMode": "code",
+              "exemplar": true,
+              "expr": "sum(watcher_workqueue_depth{container='tekton-chains-controller',app='tekton-chains-controller'})",
```

Comment:
I validated this query works in prod-rh01 from the OCP metrics console.

In theory you should have permissions to do that as well @jkhelil 

If you get a sec, try logging into the OCP console using the red hat SSO and try your query over at https://console-openshift-console.apps.stone-prd-rh01.pg1f.p1.openshiftapps.com/monitoring/query-browser?query0=